src/syslog_mcp/server.py

```
@@ -23,6 +30,11 @@ def create_server() -> FastMCP:
     
     # Register device analysis tools
     register_device_analysis_tools(mcp)
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)